packages/odyssey-react/src/components/Text/Text.theme.ts

```diff
@@ -14,11 +14,13 @@ import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
   // Font
+  /* eslint-disable @okta/odyssey/no-invalid-theme-properties */
   ColorTextBody: theme.ColorTextBody,
```

Comment:
Which rules are these violating?

Reply:
@edburyenegren-okta the `filename` rule since they contain `Text` 🙄